core/src/main/java/org/aerogear/mobile/core/ServiceModule.java

```diff
@@ -26,10 +26,4 @@
      * @return <code>true</code> if the singleThreadService configuration should be defined.
      */
     boolean requiresConfiguration();
-
```

Comment:
Does this change belong in this PR?

Reply:
Yes. As I have deleted the service cache, there is no way to find instances of services and destroying them. Thus, destroy method was obsolete. All impls of that method was empty anyway.